Dockerfile

```diff
@@ -39,7 +39,7 @@
 #   | Jul 2021 | 16.5.0 | 7.19.1 |
 
 # Note: IF YOU CHANGE THIS, change the '-nodeXX' suffix within 'build' script.
-FROM node:18-alpine
+FROM --platform=linux/amd64 node:18-alpine
```

Comment:
As noted in the description, the platform is more of a convenience to avoid problem with those building using Apple Silicon

Reply:
Hi @khenneuse 

Noticed this - by sheet luck - only today. I’ve moved on from Firebase but appreciate you sending a PR.